File: WPF客户端/AnnHome/AnnHome/Entity/Datas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AnnHome.Entity
{
    internal class Datas
    {
        private string status;
        private int count;
        private int count_total;
        private int pages;
        private List<Posts> posts;

        public string Status
        {
            get
            {
                return status;
            }

            set
            {
                status = value;
            }
        }

        public int Count
        {
            get
            {
                return count;
            }

            set
            {
                count = value;
            }
        }

        public int Count_total
        {
            get
            {
                return count_total;
            }

            set
            {
                count_total = value;
            }
        }

        public int Pages
        {
            get
            {
                return pages;
            }

            set
            {
                pages = value;
            }
        }

        public List<Posts> Posts
        {
            get
            {
                return posts;
            }

            set
            {
                posts = value;
            }
        }
    }
}

File: Android客户端/AnnHome/app/src/main/java/entity/Datas.java
package entity;

import com.google.gson.annotations.Expose;

import java.io.Serializable;
import java.util.List;

/**
 * Created by mrpan on 15/11/17.
 */
public class Datas extends BaseEntity {

    @Expose
    private String status;
    @Expose
    private int count;
    @Expose
    private int count_total;
    @Expose
    private int pages;
    @Expose
    private List<Posts> posts;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public int getCount_total() {
        return count_total;
    }

    public void setCount_total(int count_total) {
        this.count_total = count_total;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public int getPages() {
        return pages;
    }

    public void setPages(int pages) {
        this.pages = pages;
    }

    public List<Posts> getPosts() {
        return posts;
    }

    public void setPosts(List<Posts> posts) {
        this.posts = posts;
    }
}
